packages/uploadthing/src/sdk/index.ts

```diff
@@ -55,8 +60,39 @@ export const getFileUrls = async (fileKeys: string[] | string) => {
     },
     body: JSON.stringify({ fileKeys }),
   });
+
   if (!res.ok) {
     throw new Error("Failed to get file urls");
   }
+
   return res.json().then(({ data }) => data as { key: string; url: string }[]);
 };
+
+/**
+ * Request file list from UploadThing storage.
+ *
+ * @example
+ * const data = await listFiles();
+ * console.log(data); // {fileKeys: ["2e0fdb64-9957-4262-8e45-f372ba903ac8_image.jpg"], fileIds: ["2e0fdb64-9957-4262-8e45-f372ba903ac8"]}
+ */
+export const listFiles = async () => {
+  if (!UT_SECRET) throw new Error("Missing UPLOADTHING_SECRET env variable.");
+
+  // TODO: Implement filtering and pagination
+  const res = await fetch(generateUploadThingURL("/api/listFiles"), {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      "x-uploadthing-api-key": UT_SECRET,
+      "x-uploadthing-version": UPLOADTHING_VERSION,
+    },
+  });
+
+  if (!res.ok) {
+    throw new Error("Failed to list files");
+  }
+
+  return res.json().then(({ data: { files } }) => {
```

Comment:
This doesn't match the response from the infra

```suggestion
  return res.json().then(({ files }) => {
```

Reply:
<img width="567" alt="CleanShot 2023-07-17 at 23 40 23@2x" src="https://github.com/pingdotgg/uploadthing/assets/51714798/b623764f-8fa3-498e-a052-bc5c405b4672">